### ISSUE TITLE
Retrieve connection options from pgp instance

### ISSUE THREAD

**Issue by stepankuzmin** [vitaly-t/pg-promise#322]
Hi there! Is there any way to retrieve database connection options from pgp instance? For example something like this:

```js
const pgp = require('pg-promise')();
const db = pgp(connection);

db.$config.connection
```

**Comment by vitaly-t** [comment]
Not presently. And nobody ever asked about it before. It wouldn't be of much value though, considering that you are passing the connection into the database object yourself anyway.

**Comment by stepankuzmin** [comment]
I agree, but it makes sense if you make reusable library wich accepts `db` as a parameter and needs information about connection itself. We can pass the connection object into the library too, though.

**Comment by vitaly-t** [comment]
>  if you make reusable library wich accepts db as a parameter and needs information about connection itself

Valid point, marking it as a feature request.

**Comment by stepankuzmin** [comment]
Great! I think that it wouldn't be hard to implement. Maybe I can make a PR for this?

**Comment by vitaly-t** [comment]
No worries, I will implement it soon. Will require tests + documentation updates, so I better do it myself ;)

**Comment by vitaly-t** [comment]
Implemented in [v5.6.8](https://github.com/vitaly-t/pg-promise/releases/tag/v.5.6.8).

**Comment by vitaly-t** [comment]
In [version 6.x](https://github.com/vitaly-t/pg-promise/tree/6.x) of the library, [Database](http://vitaly-t.github.io/pg-promise/Database.html) also exposes property [$pool](http://vitaly-t.github.io/pg-promise/Database.html#.$pool), which represents the [connection pool](https://github.com/brianc/node-pg-pool) used by the object.
